modules/dns/dns_api.rb

```diff
@@ -73,6 +78,14 @@ class Api < ::Sinatra::Base
       end
     end
 
+    def validate_srv_value!(value)
+      priority,weight,port,target = value.split(' ')
+      validate_dns_name!(target)
+      raise Proxy::Dns::Error.new("Invalid DNS SRV value #{value}") unless priority.to_i >= 0 and
+        weight.to_i >= 0 and
```

Comment:
Both weight and priority have a rage of 0-65535 according to [RFC2782](https://tools.ietf.org/html/rfc2782)